library/proc_macro/src/lib.rs

```diff
@@ -22,7 +22,7 @@
 #![feature(staged_api)]
 #![feature(allow_internal_unstable)]
 #![feature(decl_macro)]
-#![feature(maybe_uninit_write_slice)]
+#![cfg_attr(bootstrap, feature(maybe_uninit_write_slice))]
```

Comment:
Is this actually required? I thought we stopped needing most of these after the latest bootstrap reworks…

Reply:
Hmm, that's extra strange considering the weird relative library path, but it is what it is. Thanks for verifying.

Reply:
Yes, proc-macros can be one of those special cases, because see

https://github.com/rust-lang/rust/blob/6e41e6197735bcb06c71c6d7a46de13119138578/compiler/rustc_proc_macro/Cargo.toml#L1-L5

So I believe `library/proc_macro` can still need cfg-bootstrap bits, but other parts of library should (generally) not need it anymore.